domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/StatementOfMeans.java

```diff
@@ -51,6 +51,14 @@
     @EachNotNull
     private final List<CourtOrder> courtOrders;
 
+    //    @NotNull
+    private final LivingPartner partner;
```

Comment:
why there are commented, either needed or not needed, if nullable, then get should be optional